pkgs/development/libraries/litehtml/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, gumbo
+}:
+
+stdenv.mkDerivation rec {
+  pname = "litehtml";
+  version = "0.6";
+
+  src = fetchFromGitHub {
+    owner = "litehtml";
+    repo = "litehtml";
+    rev = "v${version}";
+    hash = "sha256-9571d3k8RkzEpMWPuIejZ7njLmYstSwFUaSqT3sk6uQ=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    gumbo
+  ];
+
+  cmakeFlags = [
+    "-DEXTERNAL_GUMBO=ON"
+  ];
+
+  meta = with lib; {
+    dscription = "Fast and lightweight HTML/CSS rendering engine";
```

Comment:
```suggestion
    description = "Fast and lightweight HTML/CSS rendering engine";
```

Reply:
I already fixed that in the last force push, I wonder why ofborg didn't pick it up

@GrahamcOfBorg eval

Reply:
oh, i see. i fixed it in one package but i also copypasted the typo in the other one. fixed.